tests/sentry/replays/test_organization_selector_index.py

```diff
@@ -95,7 +95,7 @@ def test_get_replays(self):
             assert response_data["data"][0]["project_id"] == project.id
             assert (
                 response_data["data"][0]["dom_element"]
-                == 'div#myid.class1.class2[role="button"][alt="Alt"][testid="1"][aria="AriaLabel"][title="MyTitle"]'
+                == 'SignUpForm[role="button"][alt="Alt"][testid="1"][aria="AriaLabel"][title="MyTitle"]'
```

Comment:
looks like `tag` along with `id` and `class` attrs are gone, replaced by the component name. other attrs remain.

Reply:
yes if the component name exists, we want to show the component name everywhere instead of the tag, id and class since it's more readable and descriptive. This also makes it so that you can search the component name in dead and rage selectors.

Reply:
We might want to leave `id` in there, it's generally pretty descriptive/readable and not autogenerated like class